R/mobr.R

```diff
@@ -427,7 +427,305 @@ samp_ssad = function(comm, groups){
   return(comm_out)
 }
 
+# Convert specified columns of a dataframe from factors to numeric
+df_factor_to_numeric = function(dataframe, cols = NULL){
+    if (is.null(cols)) cols = 1:ncol(dataframe)
+    for (col in cols){
+        if ('factor' %in% class(dataframe[, col]))
```

Comment:
oh but I guess some objects do have more than one class so yea this is the way to go.